### DISCUSSION TITLE
Data error - Please help!

### DISCUSSION THREAD

**Discussion by stempest123** [opengisch/QField#4105] [Bugs]
Hello all,

Please could someone help with my issue. I collected data in QField app on Monday. I synchonized data on Tuesday morning but then all points disappeared on the app. Logging into QField desktop I can see that the points are there, however they all have an error and message for 'no geometry modified'. When in jobs, the error shows as 'The modified feature pk valued does not match "sourcePk" in the delta'. Would be very grateful if someone could help me retrieve my data.

### Screenshots and GIFs
![image](https://user-images.githubusercontent.com/128692753/227170028-fbeb2250-c7ee-4d85-a64c-ef777c294503.png)
![image](https://user-images.githubusercontent.com/128692753/227169902-a14ecf8b-cf6f-4b8f-a604-8727d9e68a47.png)
![image](https://user-images.githubusercontent.com/128692753/227169760-35eddf17-0699-463f-8d4f-7758db716d38.png)

### Desktop (please complete the following information)
 - OS: Windows
 - QGIS Version 3.16.3
 - QFieldSync Version ?

### Mobile (please complete the following information)
 - Device: Galaxy Tab S7
 - OS: Android 13
 - QField version: 2.7.1

If the problem happened with QFieldCloud, please add your username and project name - stempest1/Lyd_Walkover

**Comment by mbernasocchi** [comment]
This is due to the user being over the maximum storage quota.
Details in #4106
